Sources/XCTest/Private/PerformanceMeter.swift

```diff
@@ -11,7 +11,7 @@
 //  Measures the performance of a block of code and reports the results.
 //
 
-#if os(Linux) || os(FreeBSD)
+#if USE_IMPORT_FOUNDATION
```

Comment:
Corelibs XCTest is really intended to build on top of Corelibs Foundation. If it works with Darwin Foundation, that's great, but it's not a core configuration for the project. I don't think we should go out of our way to add support for that as a part of this PR, at least.